File: server/DataCore/invd.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace inv19.models { 
	/* invd -  Справочник */ 

 public class  invd_dep { // Отдел
	 public System.Guid  invd_depId{ get; set; } // Идентификатор (первичный ключ)
	[Required]
	public string  name{ get; set; } // Название
 }

 public class  invd_machine { // Машина
	 public System.Guid  invd_machineId{ get; set; } // Идентификатор (первичный ключ)
	[Required]
	public string  name{ get; set; } // Название
 }

 public class  invd_op { // Операции
	 public System.Guid  invd_opId{ get; set; } // Идентификатор (первичный ключ)
	[Required]
	public string  name{ get; set; } // Название
 }

 public class  invd_store { // Склад
	 public System.Guid  invd_storeId{ get; set; } // Идентификатор (первичный ключ)
	[Required]
	public string  name{ get; set; } // Название
 }

 public class  invd_zone { // Зона склада
	 public System.Guid  invd_zoneId{ get; set; } // Идентификатор (первичный ключ)
	[Required]
	public string  name{ get; set; } // Название
 }
}

File: client/src/app/invd.ts
import { enums } from './enums';

export namespace invd { 
	/* invd -  Справочник */ 

 export interface   invd_dep { // Отдел
	invd_depId:string; // Primary key field
	name:string; // Название
 }

 export interface   invd_machine { // Машина
	invd_machineId:string; // Primary key field
	name:string; // Название
 }

 export interface   invd_op { // Операции
	invd_opId:string; // Primary key field
	name:string; // Название
 }

 export interface   invd_store { // Склад
	invd_storeId:string; // Primary key field
	name:string; // Название
 }

 export interface   invd_zone { // Зона склада
	invd_zoneId:string; // Primary key field
	name:string; // Название
 }
}